bin/latexml

```diff
@@ -194,6 +194,7 @@ latexml [options] I<texfile>
                     by default it avoids this.
  --path=dir         adds to the paths searched for files,
                     modules, etc;
+ --log=file         specifies log file (default stderr)
```

Comment:
Your perfectly reasonable assumption just goes to show why it needed documentation!
The default is actually `<jobname>.latexml.log`.  And, the log file gets a "full" log, whereas stderr now only gets a brief report.

Reply:
I just force-pushed a change to the usage message. I think the default *used* to be stderr, but a recent upgrade changed things. Anyway, that is what led me to revise my build system, which led me to the documentation omission.